assets/js/atcb.js

```diff
@@ -542,6 +542,10 @@ function atcb_generate_google(data) {
   if (data['description'] != null && data['description'] != '') {
     url += '&details=' + encodeURIComponent(data['description']);
   }
+  if (data['recurrence'] != null && data['recurrence'] != '') {
+    url += '&recur=' + encodeURIComponent(data['recurrence']);
```

Comment:
Codacy has a fix for the issue: [Replace `'&recur='·+·encodeURIComponent(data['recurrence'` with `"&recur="·+·encodeURIComponent(data["recurrence"`](https://app.codacy.com/gh/jekuer/add-to-calendar-button/pullRequest?prid=9703945)
```suggestion
    url += "&recur=" + encodeURIComponent(data["recurrence"]);
```

Reply:
Are these Codacy fixes ignoring project preferences for single quotes?

Reply:
Actually, there were no explicit project preferences.
I started to add respective linting rules to keep this clean.
Codacy ran on the main branch by accident.
I just merged the clean-up branch (v1.9.0), which now holds all the stuff to keep code style clean and stable.